src/network/networkprotocol.h

```diff
@@ -246,6 +246,12 @@ enum ToClientCommand
 	/*
 		Signals client that sudo mode auth failed.
 	*/
+	TOCLIENT_FOV = 0x07,
+	/*
+		Sends an FOV value to client.
```

Comment:
`Send a FOV value to the player`

Reply:
`an FOV value` is the phonetically correct alternative. Also, L247 right above also uses present tense (`Signals`) instead of an imperative tone (`Signal`). Should that be changed too?

Reply:
Well, then. Keep it as-is.

Reply:
"A field of view" is correct, using 'an' with a non-vowel is bad English.

Reply:
"Signals to client that sudo mode auth failed." is clearer.

Reply:
Sorry for this minor nit-pick, but articles like 'a' and 'an' are used on the basis of phonetic vowels. In the case, "FOV" sounds like _eff-oh-vee_ and since the first sound is a vowel sound, 'an' is used here. :)

e.g. "I'll reach my home _an_ hour later"

Reply:
See [this relevant question](https://ell.stackexchange.com/questions/86665/use-of-an-for-a-word-not-starting-with-a-vowel) on ell.stackexchange.com